src/components/UsersFollowingPage.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import TopBarProgress from 'react-topbar-progress-indicator';
+import PropTypes from 'prop-types';
+import {
+  getAllUsersFollowed
+} from '../redux/actions/userFollowActions';
+import UserFollowSpan from './common/userFollowSpan';
+
+
+class UserFollowingPage extends Component {
+  componentDidMount() {
+    const { auth: { isVerified } } = this.props;
+    if (isVerified) this.props.getAllUsersFollowed();
+  }
+
+  render() {
+    let followingView;
+
+    const { userFollow: { isUsersFollowedLoading: isLoading, UsersFollowed } } = this.props;
+
+    if (!isLoading && !UsersFollowed.length) {
```

Comment:
Easier to read this as `!(isLoading || UsersFollowed.length)` => De Morgan's laws